camel-openmrs/src/main/java/org/openmrs/eip/component/service/TableToSyncEnum.java

```diff
@@ -1,100 +1,13 @@
 package org.openmrs.eip.component.service;
 
+import org.openmrs.eip.component.entity.*;
```

Comment:
Don't use wildcard imports

Reply:
Removed the use of wildcard imports and added the import for each class. There were some IDE configurations leading to this.